docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst

```diff
@@ -0,0 +1,112 @@
+Add the power of Apache Flink® to Apache Kafka® for stateful stream processing
```

Comment:
This sounds like a marketing headline. Let's just describe what this is, something like "Apache Kafka as source and sink for Apache Flink job"